cfme/tests/optimize/test_bottlenecks.py

```diff
@@ -107,4 +107,50 @@ def test_bottlenecks_time_zome(temp_appliance_extended_db, db_restore, db_tbl, d
         # Changing time zone
         view.report.time_zone.fill('(GMT-04:00) La Paz')
         row = view.report.event_details[0]
-        assert row[0].text == (db_row[0][0] - timedelta(hours=4)).strftime("%m/%d/%y %H:%M:%S BOT")
+        assert row[0].text == (db_row[0][0] - timedelta(hours=4)).strftime("%m/%d/%y %H:%M:%S -04")
+
+
+@pytest.mark.tier(2)
+def test_bottlenecks_summary_event_groups(temp_appliance_extended_db, db_restore, db_tbl,
+                                          db_events):
+    with temp_appliance_extended_db:
+        view = navigate_to(Bottlenecks, 'All')
+        # Enabling this option to show all possible values
+        view.summary.show_host_events.fill(True)
+        view.summary.event_groups.fill('Capacity')
+        events = view.summary.chart.get_events()
+        # Compare number of events in chart with number of rows in db
+        assert len(events) == db_events.filter(db_tbl.event_type == 'DiskUsage').count()
+        view.summary.event_groups.fill('Utilization')
+        events = view.summary.chart.get_events()
+        assert len(events) == db_events.filter(db_tbl.event_type != 'DiskUsage').count()
+
+
+@pytest.mark.tier(2)
+def test_bottlenecks_summary_show_host_events(temp_appliance_extended_db, db_restore, db_events):
+    with temp_appliance_extended_db:
+        view = navigate_to(Bottlenecks, 'All')
+        view.summary.show_host_events.fill(False)
+        # Checking that events with value 'Host / Node' absent in table
+        events = view.summary.chart.get_events()
+        assert not sum(1 for event in events if event.type == 'Host')
+        view.summary.show_host_events.fill(True)
+        events = view.summary.chart.get_events()
+        # Compare number of events in chart with number of rows in db
+        assert len(events) == db_events.count()
+
+
+@pytest.mark.tier(2)
+def test_bottlenecks_summary_time_zome(temp_appliance_extended_db, db_restore, db_tbl, db_events):
+    with temp_appliance_extended_db:
+        view = navigate_to(Bottlenecks, 'All')
+        events = view.summary.chart.get_events()
+        # Selecting row by uniq value
+        db_row = db_events.filter(db_tbl.message == events[0].message)
+        # Compare event timestamp with db
+        assert events[0].time_stamp == db_row[0][0].strftime("%Y-%m-%d %H:%M:%S UTC")
```

Comment:
I don't plan to insist. But consider using variable with meaningful name when you use complex constructions like db_row[0][0] more than 2 times.
here you use for example db_row[0][0] 3 times. 
btw, variable db_row[0][0] doesn't bring any meaning about what is in it. So, when you see it first time in code (assignment statement) it is more or less obvious but later it looses its meaning especially if there are several variables with similar names.